### PR TITLE
init plugin

### PR DESCRIPTION
**Pull request by nikpivkin**

### CODE REVIEW

File: Makefile
Comment (afdesk) [inline Makefile:1]: do we really need this Makefile here?

File: cmd/trivy-gitlab/main.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		os.Exit(1)
+	}
+}
+
+func NewRootCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "",
+		Short: "",
+	}
+}
```
Comment (afdesk) [inline cmd/trivy-gitlab/main.go]: why do we need this function?

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module github.com/afdesk/trivy-gitlab
+
+go 1.20
```
Comment (afdesk) [inline go.mod]: let's use 1.19

**Review by afdesk — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (nikpivkin) [reply go.mod]: done

**Pull request merged**